server/main.go

```diff
@@ -215,5 +219,7 @@ func main() {
 	reaper := newReaper(nodes, 1*time.Hour)
 	go reaper.run()
 
-	setupAndServe()
+	port := 1234
```

Comment:
Is this stray? Is there some weird scoping thing I don't understand?

Reply:
I should have been clearer - I wanted to make it easier to change the server port, since `8080` (`goji`'s default) is often already in use by other programs. One day this var will become a config or command line value.

Reply:
oh, I just meant I think this shadows the port argument to this function? but maybe I'm confused.